src/components/interface/BorrowTable/components/BorrowTableRow/components/BorrowVariableAPY/BorrowVariableAPY.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import TableCell from '@mui/material/TableCell';
+
+import { useBorrowAMMContext } from '@contexts';
+import { Typography } from '@components/atomic';
+
+const BorrowVariableAPY: React.FunctionComponent = () => {
+  const { variableApy } = useBorrowAMMContext();
+  const { result, loading, call } = variableApy;
```

Comment:
does this variable apy come directly from aave/compound or from our rate oracle similar to how it's done in the trading pool screen?